core/src/main/resources/jenkins/model/Jenkins/manage.jelly

```diff
@@ -51,6 +51,7 @@ THE SOFTWARE.
     <j:forEach var="m" items="${it.managementLinks}">
       <l:hasPermission permission="${m.requiredPermission}">
         <j:set var="icon" value="${m.iconClassName != null ? m.iconClassName : m.iconFileName}" />
```

Comment:
`getIconFileName` is `@CheckForNull`, so the line below could through NPEs I think.